pkg/agent/agent_linux.go

```diff
@@ -129,11 +129,16 @@ func (i *Initializer) ConnectUplinkToOVSBridge() error {
 	externalIDs := map[string]interface{}{
 		interfacestore.AntreaInterfaceTypeKey: interfacestore.AntreaHost,
 	}
+	// We request the same MTU for the bridge interface as for the uplink adapter. If we don't,
+	// OVS will default to the lowest MTU among all existing bridge ports, including container
+	// ports. There may be some existing workloads with a lower MTU, and using that lower value
+	// may impact host connectivity.
```

Comment:
Shall we add additional comment that big packet will be fragmented when it passthrough lower MTU interfaces (e.g. old antrea-gw0)?

Reply:
We always update the MTU of antrea-gw0 to the correct value: https://github.com/antrea-io/antrea/blob/e9990f8f417190d925fe32dd61089debcf7c1810/pkg/agent/agent.go#L698

Do you have another case in mind?